.github/issue_template.md

```diff
@@ -1,3 +1,11 @@
+# Informações importantes
+Para que as vagas enviadas neste mural sejam categorizadas com as labels, as mesmas devem seguir as seguintes regras.
```

Comment:
Eu terminaria com `:` antes da lista, o que acha?

Reply:
Faz sentido, to ajustando :)